[2025-02-13T21:02:52.084+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T21:02:52.098+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-12-17T19:17:00+00:00 [queued]>
[2025-02-13T21:02:52.105+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-12-17T19:17:00+00:00 [queued]>
[2025-02-13T21:02:52.106+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-13T21:02:52.136+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-12-17 19:17:00+00:00
[2025-02-13T21:02:52.141+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5632) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T21:02:52.142+0000] {standard_task_runner.py:72} INFO - Started process 5638 to run task
[2025-02-13T21:02:52.141+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-12-17T19:17:00+00:00', '--job-id', '15363', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpsib5op2g']
[2025-02-13T21:02:52.143+0000] {standard_task_runner.py:105} INFO - Job 15363: Subtask fetch_and_store_crypto_data
[2025-02-13T21:02:52.176+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-12-17T19:17:00+00:00 [running]> on host c6c2d3336250
[2025-02-13T21:02:52.248+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-17T19:17:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-17T19:17:00+00:00'
[2025-02-13T21:02:52.249+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T21:02:52.316+0000] {logging_mixin.py:190} INFO - Erreur lors de la récupération des données CoinGecko : 429 Client Error: Too Many Requests for url: https://api.coingecko.com/api/v3/simple/price?ids=bitcoin%2Cethereum&vs_currencies=usd
[2025-02-13T21:02:52.316+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-13T21:02:52.327+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T21:02:52.327+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-12-17T19:17:00+00:00, execution_date=20231217T191700, start_date=20250213T210252, end_date=20250213T210252
[2025-02-13T21:02:52.396+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T21:02:52.402+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
